stl/inc/flat_set

```diff
@@ -273,9 +272,7 @@ public:
         if constexpr (requires { _Mycont.append_range(_STD forward<_Rng>(_Range)); }) {
             _Mycont.append_range(_STD forward<_Rng>(_Range));
         } else {
-            for (const auto& _Val : _Range) {
-                _Mycont.insert(_Mycont.end(), _Val);
-            }
+            _Mycont.insert_range(_Mycont.end(), _STD forward<_Rng>(_Range));

```

Comment:
> However, I think we don't have to give an extra check for that chance, as it is likely that a container that doesn't support append_range will not support insert_range either.

I've changed my mind now, as I think there is no reason not to utilize a function that is required for every container by the standard.